packages/cli/src/services/project.service.ee.ts

```diff
@@ -537,4 +605,16 @@ export class ProjectService {
 	async getProjectCounts(): Promise<Record<ProjectType, number>> {
 		return await this.projectRepository.getProjectCounts();
 	}
+
+	/**
+	 * Type guard for assignable project role slugs.
+	 *
+	 * Custom project roles are supported. We consider any slug that:
+	 * - starts with the `project:` prefix, and
+	 * - is not the personal owner role
+	 * to be an assignable project role.
+	 */
+	private isAssignableProjectRoleSlug(slug: string): slug is AssignableProjectRole {
+		return slug.startsWith('project:') && slug !== PROJECT_OWNER_ROLE_SLUG;
```

Comment:
nit: Ideally this type guard would be next to the `AssignableProjectRole` type definition, so in case this is updated it visible that this should also be adapted.